starter-kits.json

```diff
@@ -3,5 +3,6 @@
   "cra-rxjs-styled-components": "Create React App, RxJS and Styled Components",
   "next-react-query-tailwind": "NextJS, React Query, and TailwindCSS",
   "remix-gql-tailwind": "Remix, GQL and TailwindCSS",
-  "vue3-apollo-quasar": "Vue3, Apollo, and Quasar"
+  "vue3-apollo-quasar": "Vue3, Apollo, and Quasar",
+  "qwik-fetch-tailwind": "Qwik, GraphQL, and TailwindCSS"
```

Comment:
We have renamed the project to `qwik-graphql-tailwind`, so we also need to update here.

Reply:
thanks @JiaLiPassion, changed!